src/20240805_Multi_RiskStewardPhase2/RiskStewardPhase2.md

```diff
@@ -1,42 +1,46 @@
+
+This AIP activates the Manual AGRS and the Automated AGRS will be activated at a later stage.
 
 ## Motivation
 
 Aave's decentralized governance system, while robust, creates significant operational overhead and cost due to its comprehensive proposal process. This process includes pre-AIP procedures, payload creation, multiple reviews, voting, and on-chain verification.
 While recent improvements have optimized some steps, further efficiency gains are possible, particularly in risk-related proposals.
 
-Rationale for reducing risk-related proposals:
+The rationale for reducing risk-related proposals:
 
-- Two expert, independent risk teams (ChaosLabs, LlamaRisk) are engaged with the DAO.
+- Two experts, and independent risk teams (ChaosLabs, LlamaRisk) are engaged with the DAO.
 - Historical voting unanimity on non-controversial, incremental risk proposals.
 - Technical feasibility of constraining updates using Aave v3 roles and specific validations.
 
 The success of the CapsPlusRiskStewards experiment demonstrates the viability of expanding this approach. As Aave expands to new networks, minimizing governance proposals becomes crucial to prevent voter fatigue.
 
 ## Specification
 
-The new RiskSteward we propose follows the same design as the CapsPlusRiskSteward, an smart contract to which the Aave Governance gives `RISK_ADMIN` role over all v3 instances, controlled by a 2-of-2 multisig, and heavily constrained on what can do and how by its own logic.
+The new Manual AGRS we propose follows the same design as the CapsPlusRiskSteward, a smart contract to which the Aave Governance gives `RISK_ADMIN` role over all v3 instances, controlled by a 2-of-2 multisig, and heavily constrained on what can do and how by its own logic.
 
-The 2-of-2 multisig controlling the RiskSteward will have Chaos and BGD Labs as signers, with the inclusion of BGD Labs as a matter of extra security procedure.
+The 2-of-2 multisig controlling the Manual AGRS will have Chaos and BGD Labs as signers, with the inclusion of BGD Labs as a matter of extra security procedure.
 
-`ACL_MANAGER.addRiskAdmin()` is called to add a the new risk steward.
+`ACL_MANAGER.addRiskAdmin()` is called to add the new Manual AGRS.
 
-Additionally, params for GHO on Arbitrum and Ethereum cannot be changed by this RiskStewards, as the params will be updated by the GhoStewards instead, this is done by setting the GHO asset as restricted via the following method: `RISK_STEWARD.setAddressRestricted();`
+Additionally, params for GHO on Arbitrum and Ethereum cannot be changed by this Manual AGRS, as the params will be updated by the GhoStewards instead, this is done by setting the GHO asset as restricted via the following method: `RISK_STEWARD.setAddressRestricted();`
```

Comment:
But this has been done in pre-setup, right?

Reply:
It is being done on the payload, as the ownership was given to the governance